server/plugin.go

```diff
@@ -130,6 +131,16 @@ func (p *Plugin) UserHasLeftTeam(pluginContext *plugin.Context, tm *model.TeamMe
 }
 
 func (p *Plugin) MessageHasBeenPosted(pluginContext *plugin.Context, post *model.Post) {
+	shouldProcessMessage, err := p.Helpers.ShouldProcessMessageNoBotCheck(post, p.api.Configurator.GetConfig().BotUserID)
+	if err != nil {
+		p.mm.Log.Error(fmt.Sprintf("Error while checking if the message should be processed %v", err))
```

Comment:
Should we use `github.com/pkg/errors.wrap` instead?

Reply:
Another option (and I think is how log is supposed to be used) is something like `p.mm.Log.Error(message, "error", err.Error())`.